### PR TITLE
return assessment in episode

### PR DESCRIPTION
**Pull request by adelaidaa**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentDto.kt [hunk 1]
```diff
@@ -2,13 +2,11 @@ package uk.gov.justice.digital.assessments.api
 
 import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.assessments.jpa.entities.assessments.AssessmentEntity
+import uk.gov.justice.digital.assessments.jpa.entities.assessments.SubjectEntity
 import java.time.LocalDateTime
 import java.util.UUID
 
-class AssessmentDto(
-
-  @Schema(description = "Assessment primary key", example = "1234")
```
Comment (adelaidaa) [inline src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentDto.kt:10]: removed the internal IDs from the exposed dtos as we don't want to exposed numeric subsequent ids for security

File: src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentDto.kt [hunk 2]
```diff
@@ -17,19 +15,39 @@ class AssessmentDto(
   val createdDate: LocalDateTime? = null,
 
   @Schema(description = "Completed Date", example = "2020-01-02T16:00:00")
-  val completedDate: LocalDateTime? = null
+  val completedDate: LocalDateTime? = null,
+
+  @Schema(description = "subject")
+  val subject: SubjectDto? = null
```
Comment (adelaidaa) [inline src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentDto.kt:21]: just a single subject. for now we will use the first or null from the assessment until we figure it out what to do with the multiple subjects

File: src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentEpisodeDto.kt [hunk 3]
```diff
@@ -9,9 +9,6 @@ import java.util.UUID
 
 data class AssessmentEpisodeDto(
 
-  @Schema(description = "Episode primary key", example = "1234")
-  val episodeId: Long? = null,
-
```
Comment (adelaidaa) [inline src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentEpisodeDto.kt:14]: remove internal numeric subsequent id for security

**Review by LucasCairns — APPROVED** [review]
LGTM 🚀

**Pull request merged**